Sources/CoreMetrics/Metrics.swift

```diff
@@ -172,14 +172,19 @@ public class Gauge: Recorder {
     }
 }
 
+public enum TimeUnit {
+    case nanoseconds, milliseconds, seconds, minutes, hours, days
+}
+
 public extension Timer {
     /// Create a new `Timer`.
     ///
     /// - parameters:
     ///     - label: The label for the `Timer`.
+    ///     - displayUnit: A hint to the backend responsible for presenting the data of the preferred display unit. This is not guaranteed to be supported by all backends.
     ///     - dimensions: The dimensions for the `Timer`.
-    convenience init(label: String, dimensions: [(String, String)] = []) {
-        let handler = MetricsSystem.factory.makeTimer(label: label, dimensions: dimensions)
+    convenience init(label: String, preferredDisplayUnit displayUnit: TimeUnit = .nanoseconds, dimensions: [(String, String)] = []) {
```

Comment:
to not be API breaking, this needs an overload that only has the existing parameters